taskfile/read/taskfile.go

```diff
@@ -141,3 +144,28 @@ func readTaskfile(file string) (*taskfile.Taskfile, error) {
 	var t taskfile.Taskfile
 	return &t, yaml.NewDecoder(f).Decode(&t)
 }
+
+func exists(dir, name string) (string, string, error) {
```

Comment:
There's a bit of code duplication in this function.

The minimum we could do here is assign the error to a global variable like we do on other places, as the same message is returned in two different places.

Bit I think a better solution would be to split this function into at least two separate functions, one with the reading part (Join + Stat calls) and the other calling the previous with the 3 tries (`entrypoint`, `"Taskfile.yml"`, and `"Taskfile.yaml"`).